tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -686,7 +686,7 @@ function test_full_sync_doesnt_send_deleted_users() {
 
 	function test_full_sync_status_with_a_small_queue() {
 
-		$this->sender->set_dequeue_max_bytes( 1500 ); // process 0.0015MB of items at a time\
+		$this->sender->set_dequeue_max_bytes( 700 ); // process 0.0007MB of items at a time
```

Comment:
The reason for that change is that the test was failing. Since now the number of bytes is smaller pre expand.